utilities/cli/util.py

```diff
@@ -542,23 +544,76 @@ def docker_args_to_devcontainer_format(docker_args: List[str]) -> List[str]:
     return result
 
 
+def get_shell_command_args(
```

Comment:
Nitpick: "shell" could refer to either host or container, suggest rename here and elsewhere

```suggestion
def get_entrypoint_command_args(
```